### ISSUE TITLE
Combining character issues - U+0E1B, U+0E49

### ISSUE THREAD

**Issue by jubby2** [MicrosoftDocs/typography-issues#940]
We have noticed a stacking problem in Angsana UPC when combining U+0E1B and U+0E49. 
<img width="314" alt="Screen Shot 2022-06-07 at 11 58 15 AM" src="https://user-images.githubusercontent.com/106703752/172451331-e6ddf4dd-e4f9-431b-9c95-711ad5aeacc3.png">

Other fonts, such as Adobe Thai, combine these characters properly:
![good character Adobe Thai](https://user-images.githubusercontent.com/106703752/172451617-11f02739-0a53-4608-a585-710cafc61ec7.png)

Is this something that can be resolved? Our corporation has 50 licenses through Fonts.com for use in InDesign, and it is our preferred font for Thai language products. 

[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d2a92229-f3a6-585f-326b-e06a5e02c341
* Version Independent ID: 60088004-f114-e8f8-8bd8-325a0d9e8b83
* Content: [AngsanaUPC font family - Typography](https://docs.microsoft.com/en-us/typography/font-list/angsanaupc)
* Content Source: [typographydocs/font-list/angsanaupc.md](https://github.com/MicrosoftDocs/typography/blob/live/typographydocs/font-list/angsanaupc.md)
* Product: **typography**
* GitHub Login: @alib-ms
* Microsoft Alias: **alib**

**Comment by alib-ms** [comment]
![image](https://user-images.githubusercontent.com/36778811/173902523-447f6359-0efe-42ed-8781-6a8e8391cbb9.png)

**Comment by tiroj** [comment]
> There are plenty of examples of the overstrike to be found in Thai lettering and printing tradition.

Yes, but it needs to be done with some care, especially when various weights of stroke are involved. It is seldom visually adequate to slap the combining mark on top of the ascender stroke as in Angsana. If the mark is going to interact with the ascender, we usually make this a ligature so that we can refine the length and weight of the stroke of the mark where it intersects the ascender. The mark may often not cut all the way across the ascender. This is Adobe Thai:
<img width="1076" alt="image" src="https://user-images.githubusercontent.com/5614500/176571494-a978c24a-4bbe-4eef-ae7c-c14a75481a01.png">

**Comment by alib-ms** [comment]
Reviewing other fonts. 
![image](https://user-images.githubusercontent.com/36778811/177657663-42b39ea4-60a2-4200-9054-4071ea743175.png)

**Comment by alib-ms** [comment]
![image](https://user-images.githubusercontent.com/36778811/177657794-fece7786-edec-43f9-94ca-810f84ade7d8.png)
![image](https://user-images.githubusercontent.com/36778811/177657819-7ba79b4f-02cc-4e28-b3a6-7f1f2f388197.png)

**Comment by alib-ms** [comment]
![image](https://user-images.githubusercontent.com/36778811/177657877-9de47815-b9d7-441a-a8f4-e109d2f86d0f.png)
